features/step_definitions/employers_steps.rb

```diff
@@ -1089,7 +1089,7 @@ def employees(*traits)
   end
 end
 
-And(/^employer clicks Create Plan Year$/) do
+And(/^employer clicks on Create Plan Year$/) do
```

Comment:
let's take the opportunity to add a page object here and reference a `data-cuke` attribute on the button